plugins/package-managers/gradle-inspector/src/main/kotlin/GradleInspector.kt

```diff
@@ -87,6 +87,11 @@ private val GRADLE_USER_HOME = Os.env["GRADLE_USER_HOME"]?.let { File(it) } ?: O
  */
 const val OPTION_GRADLE_VERSION = "gradleVersion"
 
+/**
+ * The sha1 sum for a zero by size file.
+ */
+private const val ZERO_BYTES_FILE_SHA1 = "da39a3ee5e6b4b0d3255bfef95601890afd80709"
```

Comment:
We already have this as `EMPTY_PACKAGE_VERIFICATION_CODE`. How about refactoring that first to a constant in `HashAlgorithm`?

Reply:
`EMPTY_PACKAGE_VERIFICATION_CODE` is in `spdx-utils` and does not know about the `HashAlgorithm` class. What do you think?

Reply:
It's only used in `UtilsTest` which could add the `model` as a dependency. I'll propose a follow-up refactoring.